.gitignore

```diff
@@ -7,3 +7,4 @@
 .idea/
 .vscode/
 /cmake-build-debug/
+*.o
```

Comment:
I'd rather be specific about where its ok to ignore `.o` files.

I often create temporary `.c` and `.o` file in my project directory and I want `git status` to show them to me and `git clean -f` to clean them up for me.   Maybe I'm strange like that.

Reply:
Done.